public/components/agents/vuls/inventory.tsx

```diff
@@ -51,9 +90,64 @@ export class Inventory extends Component {
     this._isMount = false;
   }
 
+  async fetchVisualizationVulnerabilitiesSummaryData(field, agentID) {
+    const results = await getAggregation(agentID, field, 4);
+    return Object.entries(results[field]).map(([key, value], index) => ({
+      label: key,
+      value,
+      color: this.colorsVisualizationVulnerabilitiesSummaryData[index],
+      onClick: () => this.onFiltersChange(this.buildFilterQuery(field, key))
+    })).sort((firstElement, secondElement) => secondElement.value - firstElement.value)
+  }
+
+  async fetchVisualizationVulnerabilitiesSeverityData(){
+    const { id } = this.props.agent;
+    const FIELD = 'severity';
+    this.setState({ isLoadingStats: true });
+    //Assign a value to the severity label to be able to sort it by level
+    const severityOrder = { Critical: 12, High: 9, Medium: 6, Low: 3 };
+
+    const vulnerabilityLastScan = await getLastScan(id);
+    const { severity } = await getAggregation(id, FIELD);
+
+    const severityStats = Object.keys(severityOrder).map(key => ({ 
```

Comment:
**nitpick**: You could define an array with the keys of severity levels instead of creating an object with values that are not used.

```ts
const severityStats = ['Critical', 'High', 'Medium', 'Low'].map(key => {/* logic */})

```